app/screens/more_dms/index.js

```diff
@@ -27,6 +27,7 @@ function mapStateToProps(state) {
         theme: getTheme(state),
         currentDisplayName: state.views.channel.displayName,
         currentUserId: getCurrentUserId(state),
+        currentUser: getCurrentUser(state),
```

Comment:
I don't think you need all the object, probably you can check if is a guest right here without passing the whole object to the component.

Reply:
Good point, I'll change it

Reply:
done